lib/ansiblelint/rules/MismatchedBracketRule.py

```diff
@@ -1,54 +0,0 @@
-# Copyright (c) 2013-2014 Will Thames <will@thames.id.au>
-#
-# Permission is hereby granted, free of charge, to any person obtaining a copy
-# of this software and associated documentation files (the "Software"), to deal
-# in the Software without restriction, including without limitation the rights
-# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-# copies of the Software, and to permit persons to whom the Software is
-# furnished to do so, subject to the following conditions:
-#
-# The above copyright notice and this permission notice shall be included in
-# all copies or substantial portions of the Software.
-#
-# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-# THE SOFTWARE.
-
-from ansiblelint import AnsibleLintRule
-
-
-class MismatchedBracketRule(AnsibleLintRule):
```

Comment:
Yeah, that's pretty much exactly it. If you try and use ansible's libraries to read a playbook with a mismatched bracket, it blows up. Given that ansible-playbook won't read it, ansible-lint no longer tries either.

Playbooks passing `ansible-playbook --syntax-check` is an undocumented requirement of ansible-lint (i.e. I don't try and fix bugs that the syntax checker doesn't like)